misc/geolite-update.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+# set -xe
+
+# geolite-update.sh -- update GeoIP lite databases.
+# (C) 2017 Ettercap Development Team.
+
+# Ettercap can use MaxMind's GeoIP databases to look up the country for an IP address.
+# This simple shell script helps in downloading and installing the *free* GeoLite
```

Comment:
Wouldn't it make sense putting here some advice, that distribution provided packages are recommended and are detected automatically? This method is IMHO targeted for systems that don't provide the possiblity to easily install GeoIP database.

Reply:
I guess you're right. I didn't notice that some distribution provide these by default.
I thought they don't since they are none-free.